packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-load-balancer.ts

```diff
@@ -307,7 +307,7 @@ export abstract class BaseLoadBalancer extends Resource {
   protected resourcePolicyPrincipal(): iam.IPrincipal {
     const region = Stack.of(this).region;
     if (Token.isUnresolved(region)) {
-      throw new Error('Region is required to enable ELBv2 access logging');
+      return new iam.ServicePrincipal('logdelivery.elasticloadbalancing.amazonaws.com');
```

Comment:
Only newer regions use this principal for elb logging ([doc](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/enable-access-logging.html#attach-bucket-policy)), and other (older) regions require the `elb-account-id` for each region.

So with this change, you _can_ synthesize an environment-agnostic template using elb access logging, but it won't work in those older regions. I guess that's not the desired result. Correct me if I misunderstand something 🙏

Reply:
I see. 

Then the proposed solution in the issue wont work. I should have done some more research here before opening PR. Thank you for review @tmokmss . Would you like to inform in the issue as well?

Reply:
Sure I commented to the original issue, and thank you for working on the cdk issues!